src/webapp/features/rabble/components/Theme.tsx

```diff
@@ -2,7 +2,9 @@ import React, { useState, useEffect } from "react";
 
 const themes = {
   dark: {
-    bg: "#3d3d29",
+    // bg: "#3d3d29",
+    // bg: "#23201c",
+    bg: "#2e353d",
```

Comment:
Also maybe these commented lines can go?  Unless we need em, up to you

Reply:
just options open! easily removable, I'll do a deep dive on the theme later, it requires going into each module and using the vars where appropriate.